docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -47,22 +41,19 @@ You should subscribe to:
 Collection infrastructure
 =========================
 
-
 The following guidelines describe the required structure for your collection:
 
-* MUST have a publicly available issue tracker that does not require a paid level of service to create an account or view issues.
+* MUST have a publicly available issue tracker that does not require a paid level of service to create an account and to create and view issues.
+* MUST have the issue feature enabled in its repository and accept issue reports from anyone.
 * MUST have a Code of Conduct (CoC).
 
-  * The collection's CoC MUST be compatible with the :ref:`code_of_conduct`.
-  * The collections SHOULD consider using the Ansible CoC if they do not have a CoC that they consider better.
-  * The :ref:`Diversity and Inclusion working group <working_group_list>` may evaluate all CoCs and object to a collection's inclusion based on the CoCs contents.
-  * The CoC MUST be linked from the ``README.md`` file, or MUST be present or linked from the ``CODE_OF_CONDUCT.md`` file in the collection root.
```

Comment:
Lost during re-formulation process:) back now, thanks!

Reply:
I'm also not sure if we need to have diversity & inclusion wg involvement here.. is it alive? could we just request a link to the Ansible CoC? It's not bad:) Also we don't have a mechanism to implement this and also their conclusion s probably don't have any legal power.
This statement imo unreasonably complicates the process. I'll remove it now.
Feedback is welcome

Reply:
We don't want to force the Ansible CoC on every organization / collection that's included. This will likely make it impossible for some organizations to get their collection included. So I guess we still need the WG, even though it seems to be pretty inactivate right now.

Reply:
It's not gonna work with the WG or w/o imo.
If there aren't lawyers or any other people with special skills to evaluate docs _compatibility_, there's no difference between us and them, so we can "evaluate" them ourselves based on our personal and very subjective feelings.
How about just requiring a CoC with a recommendation to have a link to the Ansible CoC dropping all that stuff about WG and  subjective-in-our-case compatibility?

Reply:
I'm still against just changing it as a small part of such a huge PR. This is IMO an important thing that needs its own discussion, and that discussion has to include the Diversity and Inclusion WG.

Reply:
SGTM, is leaving it as is for now fine?